sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -45,8 +45,8 @@
     and [System.Net.Http](https://learn.microsoft.com/en-in/dotnet/core/diagnostics/built-in-metrics-system-net#systemnethttp).
   * For environments targetting `.NET 7.0` and below, distro uses ASP.NET Core and HTTP Client Instrumentation.
     Detailed metrics information can be found in the
-    [ASP.NET Core Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/README.md#list-of-metrics-produced)
-    and [HTTP Client Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.Http/README.md#list-of-metrics-produced)
+    [ASP.NET Core Instrumentation](https://github.com/open-telemetry/opentelemetry-dotnet/blob/Instrumentation.AspNetCore-1.7.1/src/OpenTelemetry.Instrumentation.AspNetCore/README.md#list-of-metrics-produced)
```

Comment:
this should be 1.8.1?

Reply:
This changelog entry is for 1.2.0-beta2 (2024-03-12). I changed the link from the `main` branch to the tag that we would have been working against that month.

This is blocking other teams PRs today. Lets merge this change to unblock, and we can discuss offline if we should use something else.

Reply:
Missed the version number